source/Lexer.cpp

```diff
@@ -129,32 +131,29 @@ Token Lexer::concatenateTokens(BumpAllocator& alloc, Token left, Token right) {
     mem[newLength - 1] = '\0';
     StringRef combined{ (char*)mem, newLength };
 
-    // common case: identifier + identifier
```

Comment:
I got rid of this since two identifiers don't always form an identifier.  They can instead form a keyword.  I tested this on eda playground and those compilers do recognize keywords formed in this way.